pipeline/trusted-resources/pkg/trustedtask/verify.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trustedtask
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/sha256"
+
+	cosignsignature "github.com/sigstore/cosign/pkg/signature"
+	"github.com/sigstore/sigstore/pkg/signature"
+	"github.com/sigstore/sigstore/pkg/signature/kms"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"knative.dev/pkg/apis"
+	"knative.dev/pkg/system"
+	"knative.dev/pkg/webhook/json"
```

Comment:
Any reason to use this over `encoding/json`?

Reply:
it should be encoding/json, I guess probably my ide somehow auto import the wrong pkg